lldb/test/API/commands/platform/process/launch/TestPlatformProcessLaunch.py

```diff
@@ -2,7 +2,9 @@
 Test platform process launch.
 """
 
+import lldb
```

Comment:
Is this required?

Reply:
This is a debug artifact. Removed now. Thanks.